rest-api-with-express/app.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const { Romcal } = require('romcal');
+const { France_Fr: FranceFR } = require('@romcal/calendar.france');
```

Comment:
non-camelcase variable names are frowned upon in airbnb styleguides (what we use for eslint)

Reply:
Oh, that is strange. Does it not throw errors at other places, either in some app in this repo or in romcal/romcal? :thinking:

Reply:
it doesn't, because airbnb isn't default elsewhere. While romcal/romcal uses it in [generated code](https://github.com/romcal/romcal/blob/dev/scripts/bundle.ts#L221) or as a variable, and doesn't use airbnb; I also wouldn't want to break our i18n keys immediately, which is why I've not applied it there.

I'm trying to use fairly common and widely used styles (airbnb being the largest that i'm aware of apart from not using a styleguide at all) for the example projects to avoid using very niche or custom configurations